onnxscript/rewriter/pattern.py

```diff
@@ -8,6 +8,8 @@
 import inspect
 import itertools
 import math
+from collections import defaultdict
+from enum import IntEnum
```

Comment:
```suggestion
import enum
```

I would import modules only in general